pyproject.toml

```diff
@@ -35,6 +35,7 @@ opentelemetry-sdk = {version = "^0.13b0", optional = true}
 python-dateutil = "^2.7.0"
 cached-property = "^1.5.2"
 pydantic = {version = "^1.6.1", optional = true}
+python-multipart = "^0.0.5"
```

Comment:
Is this being used by anything?

Reply:
Yes, if you want to use multipart with starlette.

Reply:
Ah ok 👍